R/between_within_classic.R

```diff
@@ -0,0 +1,98 @@
+#' Calculation of between-within Degrees of Freedom for One-Dimensional Contrast
+#'
+#' @description Calculates the estimate, standard error, degrees of freedom,
+#' t statistic and p-value for one-dimensional contrast. Used in [df_1d()] if method is
+#' "Between-within".
```

Comment:
wait, I would expect this is used when the method is something like `Between-within-classic`?